src/label_projection/control_methods/baseline_majority_vote/test_script.py

```diff
@@ -0,0 +1,22 @@
+import subprocess
```

Comment:
If you want, you could write one unit test in src/label_projection/tests/test_method.py which can be reused for all components. Example: https://github.com/openproblems-bio/neurips2021_multimodal_viash/blob/main/src/match_modality/unit_tests/test_method.py